### ISSUE TITLE
Port in Markdown links breaks URL

### ISSUE THREAD

**Issue by snwiem** [Requarks/wiki#3063]
**Describe the bug**
Given `[http://my-server:10000](My Server)` generats a link `http://my-server\:10000`.

**To Reproduce**
1. Use the markdown syntax given above on any page within WikiJS

**Expected behavior**
The expected rendered link is `http://my-server:10000`

**Host Info (please complete the following information):**
 - OS:
```snwiem@debian-server:~$ uname -a
Linux debian-server 4.9.0-14-amd64 #1 SMP Debian 4.9.240-2 (2020-10-30) x86_64 GNU/Linux
```
- Docker
```
snwiem@debian-server:~$ docker --version
Docker version 19.03.15, build 99e3ed8919
```
- Image
![grafik](https://user-images.githubusercontent.com/5032664/107582287-95409b00-6bf9-11eb-8fb3-32cf135e34b2.png)

 - Wiki.js version: [e.g. 2.0.1]
![grafik](https://user-images.githubusercontent.com/5032664/107582144-61657580-6bf9-11eb-81a6-974dfca74c4c.png)

 - Database engine: [e.g. postgres 9.7, mysql 5.7]
![grafik](https://user-images.githubusercontent.com/5032664/107582417-bd2ffe80-6bf9-11eb-888d-bd3e9b179727.png)

**Comment by mudfly** [comment]
Hi, @snwiem I am not a member of this project but was clicking through bugs. It looks to me like your markdown is backward. [link text](link URL). I've tested adding a port number to a URL constructed in this way, and it works as expected.

**Comment by NGPixel** [comment]
Your markdown is indeed backwards.

```
[TITLE](URL)
```

**Comment by snwiem** [comment]
First of all thank you for this fast reply. I'm sorry. I totally mixed the syntax up when posting this issue.
Within Wiki.js I used the correct syntax.
After carefully reviewing my markdown after your comments, I noticed that it seems that the fully qualified link-urls are handled as internal links when using **the same host name as the one wiki.js is running on**.

The hostname where my Wiki.js instance is hosted is named `debian-server`.

I attached to screenshots from the life system which should illustrate this.

![wiki_syntax](https://user-images.githubusercontent.com/5032664/107658380-5e649680-6c86-11eb-8a9c-4cd1a8a2d332.PNG)
![internal_link](https://user-images.githubusercontent.com/5032664/107658393-6290b400-6c86-11eb-9b5b-cde24d912454.PNG)

Is there a way to prevent wiki.js to process the host names ?

I also tried to use pure HTML within the markdown page

```html
<a href="http://debian-server:8081">HTML Link</a>
```

but it seems to be processed to

```html
<a href="/:8081" class="is-internal-link is-invalid-page">HTML Link</a>
```

also.

Best regards
Sebastian
